e2e/tests/stats-page.spec.ts

```diff
@@ -22,7 +22,7 @@ test.beforeEach(async ({page}) => {
 });
 
 test('matches the screenshot', async ({page}) => {
-  await page.goto('http://localhost:5555/stats');
+  await page.goto('http://localhost:5555/stats?showFeaturesLagging');
```

Comment:
Move this to a separate test

https://github.com/GoogleChrome/webstatus.dev/blob/eec191ba897b9219b8863dd4b1440881716e9700/e2e/tests/feature-page.spec.ts#L40-L58